core/src/com/unciv/logic/trade/TradeEvaluation.kt

```diff
@@ -158,6 +162,16 @@ class TradeEvaluation {
             }
         }
     }
+    fun isSurroundedByOurCities(city: CityInfo, civInfo: CivilizationInfo): Int{
+        val borderingCivs: ArrayList<String> = city.cityBorders()
+        if (borderingCivs.size == 1 && borderingCivs.asSequence().contains(civInfo.civName)){
+            return 10*civInfo.getEraNumber() // if the city is surrounded only by trading civ
+        }
+        if (borderingCivs.asSequence().contains(civInfo.civName))
```

Comment:
No need to .asSequence() this if you're only going to check .contains()

Reply:
K will make change

Reply:
fixed